### ISSUE TITLE
Portable Text Editor: Dropdowns in inline reference popovers are cut off

### ISSUE THREAD

**Issue by basharalidreesi** [sanity-io/sanity#4491]
**Describe the bug**

The functionality of inline references in the Portable Text Editor (as demonstrated in [a guide by Saskia Bobinska](https://www.sanity.io/guides/add-inline-blocks-to-portable-text-editor)), while immensely useful, is rendered ineffective by a limitation in the height of the reference field's autocomplete dropdown.

**To reproduce**

1. Follow this guide: https://www.sanity.io/guides/add-inline-blocks-to-portable-text-editor
2. Add a new inline reference to the Portable Text Editor
3. Attempt to select a document using the reference field's dropdown selector
4. Notice that only a sliver of the dropdown selector (about 10px) is visible
5. Notice that you can still scroll within the aforementioned sliver to find your desired document

**Expected behavior**

A more expansive view of the reference field's dropdown selector.

**Screenshot**

<img width="636" alt="Screenshot 2023-05-14 at 13 18 42" src="https://github.com/sanity-io/sanity/assets/69329190/405024c2-74c4-443e-8a31-e70e2f24c14c">

**Which versions of Sanity are you using?**

```
@sanity/cli (global)          3.10.0 (latest: 3.10.3)
@sanity/eslint-config-studio  2.0.1 (up to date)
@sanity/icons                 2.3.1 (up to date)
@sanity/vision                3.10.3 (up to date)
sanity                        3.10.3 (up to date)
```

**What operating system are you using?**

macOS Ventura 13.3.1

**Which versions of Node.js / npm are you running?**

```
9.6.4
v19.9.0
```

**Comment by basharalidreesi** [comment]
@adam-rocska None yet! I wonder if a custom component could help alleviate this problem until a fix comes. I will post here if I fin a solution.

**Comment by basharalidreesi** [comment]
This has been fixed in [v3.11.2](https://github.com/sanity-io/sanity/releases/tag/v3.11.2).
